ContosoCrafts.WebSite/Services/JsonFileCategoryService.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text.Json;
+using Microsoft.AspNetCore.Hosting;
+using ContosoCrafts.WebSite.Models;
+
+namespace ContosoCrafts.WebSite.Services
```

Comment:
There should already be a method inside of JsonFileMenuService to deserialize the category json file - If you would like to use refactor the code then we can reuse what is in JsonFileMenuService or if you feel it's more appropriate you can choose to remove the GetCategories method from inside of the JsonFileMenuService